drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -100,7 +100,7 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     /* set default TX power */
     at86rf2xx_set_txpower(dev, AT86RF2XX_DEFAULT_TXPOWER);
     /* set default options */
-    at86rf2xx_set_option(dev, NETDEV2_IEEE802154_PAN_COMP, true);
+    at86rf2xx_set_option(dev, NETDEV2_IEEE802154_PAN_COMP, false);
```

Comment:
@miri64 I think you're right. There is no explicit implementation behind this option, so simply removing this line would probably be the most logical thing?